Source/Core/Core/PowerPC/JitCommon/JitBase.cpp

```diff
@@ -12,6 +16,30 @@
 #include "Core/PowerPC/PowerPC.h"
 #include "Core/System.h"
 
+const std::vector<std::pair<bool JitBase::*, const Config::Info<bool>*>> JitBase::JIT_SETTINGS = {
```

Comment:
I'd use a std::array, rather than a std::vector here, so no heap allocation is required for these